internal/compiler/builtins.slint

```diff
@@ -415,7 +415,21 @@ PropertyAnimation := _ {
 
 export struct StandardListViewItem := {
     //-name:slint::private_api::StandardListViewItem
-    text: string
+    text: string,
+    editable: bool,
```

Comment:
That's good.  But we should also support editing in the StandardListView.
This can be done in separate commit, but  wonder if we shouldn't even do that first.

Reply:
@ogoffart a little bit easier would be to merge that first and as next topic to add that feature to the ListView. I would be ready to do it next.